### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by boba-and-beer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,117 @@
...
+| ------------- | ----------- |
+| Rest API      | [Documentation](https://docs.relevance.ai/docs/quickstart) | 
+| SDK     | [Documentation](https://relevanceai.github.io/RelevanceAI/docs/html/index.html)        |
+
+# Development
+
+## Getting Started
+To get started with development, ensure you have pytest and mypy installed. These will help ensure typechecking and testing.
+
+    python -m pip install RelevanceAi[dev]
```
Comment (charyeezy) [inline README.md:84]: ```python -m pip install RelevanceAI[dev]```

File: README.md [hunk 2]
```diff
@@ -1,70 +1,117 @@
...
+The syntax for selecting an option is section.key. For example, to disable logging, run the following to modify logging.enable_logging:
 
-```python
-client.config.set_option('logging.enable_logging', False)
-```
+    client.config.set_option('logging.enable_logging', False)
 
 To restore all options to their default, run the following:
 
+<<<<<<< HEAD
```
Comment (charyeezy) [inline README.md:110]: Remove these merge conflicts here ! :)

File: README.md [hunk 3]
```diff
@@ -1,70 +1,117 @@
...
+    client.config.reset_to_default()
+
+
+=======
```
Comment (charyeezy) [inline README.md:114]: Here too!

File: README.md [hunk 4]
```diff
@@ -82,3 +129,4 @@ You can also update the ingest base URL:
 ```
 client.ingest_base_url = "https://.../latest
 ```
+>>>>>>> main
```
Comment (charyeezy) [inline README.md:132]: here too ~

File: README.md [hunk 5]
```diff
@@ -1,70 +1,117 @@
...
+- Pay-to-use model (100.000 FREE API calls per month)
 
-You can also install on conda (only available on Linux environments at the moment): `conda install -c relevance relevanceai`.
+# Installation
 
-## How to use the RelevanceAI client
+You can install it this way via `pip` 
 
-For example:
+    !pip install -U relevanceai
```
Comment (charyeezy) [inline README.md:21]: ```!pip install RelevanceAI```

**Comment by boba-and-beer** [comment]
Closing this branch in favor of this: https://github.com/RelevanceAI/RelevanceAI/pull/193

**Pull request closed without merging**